### DISCUSSION TITLE
Keystrokes?

### DISCUSSION THREAD

**Discussion by NRKorsakov** [shanteacontrols/OpenDeck#270] [Q&A]
Is there any way for OpenDeck to send keyboard keystrokes, as in letters or numbers?

I use numpad-zero to punch in and out of record, and would like OpenDeck to be able to send that key.

Using MMC-Record disables certain stuff I use in my DAW for some reason, so I set things up this way.

Thanks!

**Comment by paradajz** [comment]
There isn't.

**Comment by NRKorsakov** [comment]
Hey thanks anyway!
